src/main/java/com/kc/project/rashmiwebservice/RashmiSet.java

```diff
@@ -49,14 +38,11 @@ public void remove(int item) {
             break;
             }
         }
-            int[] temp = new int[intArr.length-1];
-            for(int j=0,k=0;j<intArr.length;j++){
-                if(index==j || intArr[j]== item){
-                    System.out.println("count:::" +count--);
-                    continue;
-                }
-                temp[k++] = intArr[j];
-            }
-            intArr=temp;
+        if(index>0) {
```

Comment:
We don't need a new array here. Think about this slack message. Any updates you think you can make?
![image](https://user-images.githubusercontent.com/38619160/109845961-49d74680-7c13-11eb-8920-a78947a5546a.png)